addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -80,6 +80,8 @@ def _compute_duplicated_lead_ids(self):
     @api.depends('action')
     def _compute_partner_id(self):
         for convert in self:
+            if not convert.action:
```

Comment:
Strange, as _compute_partner_id depends on action which should be computed before.

Isn't it a 'lead_id' trigger missing on _compute_partner_id by the way ?